### PR TITLE
feat: generate client read.me

### LINKED ISSUE #5: we need to also generate a nice README.md for the client
This template should not only generate `client.py` but also a readme that explain how to use it with some sample code

### PR DESCRIPTION
**Pull request by thulieblack**
resolves #5

### CODE REVIEW

File: template/utils.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    return `${asyncapi.info().title()} Client`;
+  };
+  
+  // Function to generate service client description
+  export const getServiceClientDescription = (asyncapi) => {
+    return `${asyncapi.info().description()}`;
+  };
+  
+  // Function to import client service
+  export const getClientService = (asyncapi) => {
```
Comment (derberg) [inline template/utils.js:14]: `getClientClassName`

File: template/readme.md.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+function receiveFunctionExample(functions) {
+  let content = '';
+
+  functions.forEach(t => {
+    content += `
+# ${t.summary}
+client.${t.functionName}(read_views)
+print("Receive message: " + message.topic)
```
Comment (derberg) [inline template/readme.md.js:150]: extend with callback example and explanation please

**Comment by thulieblack** [comment]
@derberg this is ready

File: test/fixtures/asyncapi.yml [hunk 3]
```diff
@@ -54,7 +54,7 @@ components:
   schemas:
     commentCount:
       type: object
-    additionalProperties: false
+      additionalProperties: false
     properties:
```
Comment (derberg) [inline test/fixtures/asyncapi.yml:58]: I think this should also be indented to become part of `commentCount`

File: template/readme.md.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+      <Text>
+      ```
+      </Text>
+
+      <Text newLines={2}>
+        ## Usage of {serviceClientName}
+      </Text>
+
+      <Text newLines={2}>
+        - Import the {serviceClientName}:
```
Comment (derberg) [inline template/readme.md.js:63]: why you decided to change to `-` and bullet points instead of `1.` everywhere with numbered list?

Comment (derberg) [reply template/readme.md.js:63]: look at answer number 3 -> https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list-in-markdown-format

it happens when there are code blocks and also empty spaces between numbers - you just have to make sure there are no empty lines between points + make sure code blocks have proper indent

Comment (thulieblack) [reply template/readme.md.js]: Im not sure what exactly I'm doing wrong but it's not rendering, and I double-checked the indentation

File: template/readme.md.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+      <Text newLines={2}>
+        ## Usage of {serviceClientName}
+      </Text>
+
+      <Text newLines={2}>
+        1. Import the {serviceClientName}:
+      </Text>
+
+      <Text>
```
Comment (derberg) [inline template/readme.md.js:66]:
```suggestion
      <Text indent={4}>
```

**Comment by thulieblack** [comment]
Fixed lets get it merged finally 🎉

**Review by derberg — APPROVED** [review]
well done 🚀

**Pull request merged**